packages/client/tests/functional/chunking-query/tests.ts

```diff
@@ -177,7 +177,13 @@ testMatrix.setupTestSuite(
         const ids = generatedIds(EXCESS_BIND_VALUES)
 
         if (usingRelationJoins) {
-          await expect(selectWith2InFilters(ids)).rejects.toThrow(RELATION_JOINS_NO_CHUNKING_ERROR_MSG)
+          if (driverAdapter === 'js_pg' || driverAdapter === 'js_pg_cockroachdb') {
+            await expect(selectWith2InFilters(ids)).rejects.toThrow(
+              'The query parameter limit supported by your database is exceeded',
+            )
+          } else {
+            await expect(selectWith2InFilters(ids)).rejects.toThrow(RELATION_JOINS_NO_CHUNKING_ERROR_MSG)
+          }
         } else {
           await expect(selectWith2InFilters(ids)).rejects.toThrow()
         }
```

Comment:
I feel like this test should probably be using error codes and not messages. No need to change it here in this PR and this is also fine, it's just that error codes exist precisely so that error messages don't become a part of public API.